docs/04.guides/04.cookbooks/31.Externalize_Strings/page.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Externalize strings
+id: Externalizing_Strings
+---
+## Externalize strings ##
+
+Externalize strings from generated class files to separate files. This method is used to reduce the memory of the static contents for templates. We explain this method with a simple example below:
+
+**Example:**
+
+```lucee
+//index.cfm page 
+
+<cfset year = 1960>
+<html>
+	<body>
+		<h1>....</h1>
+		.......
+		.......
+		It was popular in the <cfoutput> #year# </cfoutput>
+		.......
+		<b>....</b>
+	</body>
+</html>
+```
+
+1. Here Index.cfm file contains lot of strings(static contents) inside, but there is no functionality. Just give a cfoutput with year, In year is already declared by using <cfset> in top of the Index.cfm page.  
```

Comment:
***ToChange***
Here the Index.cfm file contains a lot of strings (static contents), but there is no functionality. The file just gives a cfoutput with year. The varible string 'year' is already declared by using <cfset> in top of the Index.cfm page.